subsys/bluetooth/mesh/shell/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Bluetooth mesh shell configuration options
+
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_MESH_SHELL_HEALTH_SRV
+	bool "Support for Shell Health Server Model"
```

Comment:
Should it be default `y`?

Reply:
IMO, no. Some of the point is to make the user aware that the health server features of shell requires some initialization. If the option is set by default the command will apear in the shell menu, even though it is useless due to lacking initialization.